app/views/layouts/hyrax.html.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
Is this being overwritten from the default provided by hyrax? If so, add the tag to comment where it differs?